ios/Plugin/VoiceRecorder.swift

```diff
@@ -22,44 +27,25 @@ public class VoiceRecorder: CAPPlugin {
         }
     }
     
-    @objc public func hasAudioRecordingPermission(_ call: CAPPluginCall) {
+    @objc func hasAudioRecordingPermission(_ call: CAPPluginCall) {
         call.resolve(ResponseGenerator.fromBoolean(doesUserGaveAudioRecordingPermission()))
     }
 
-    @objc public func startRecording(_ call: CAPPluginCall) {
+    @objc func startRecording(_ call: CAPPluginCall) {
         if(!doesUserGaveAudioRecordingPermission()) {
             call.reject(Messages.MISSING_PERMISSION)
             return
         }
         
-        if(customMediaRecorder != nil) {
-            call.reject(Messages.ALREADY_RECORDING)
-            return
-        }
-        
-        customMediaRecorder = CustomMediaRecorder()
-        if(customMediaRecorder == nil) {
-            call.reject(Messages.CANNOT_RECORD_ON_THIS_PHONE)
-            return
-        }
-        
```

Comment:
these lines should probably fix unintentional syncing problems, but I'll test:
* call someone
* start recording - should fail
* finish call
* Try to record again - should start - not fail